README.md

```diff
@@ -364,6 +364,9 @@ $ cargo install scx_rusty
 
 and `scx_rusty` will be built and installed as `~/.cargo/bin/scx_rusty`.
 
+## SystemD services
```

Comment:
nit: It looks a bit odd to me to say "SystemD" -- can we just stick with "`systemd`"?

Reply:
Fixed!